devtools/ci/gh-actions/conda-envs/build-M1-arm64.yml

```diff
@@ -0,0 +1,19 @@
+name: build
+channels:
+- conda-forge
+- bioconda
+dependencies:
+# build
+- cmake
+- ccache
+# host
+- python
+- cython
+- swig
+- fftw
+- numpy
+- doxygen
```

Comment:
Let's pin doxygen to 1.9.1, which is the current latest version.  Otherwise there's a high chance of the build spontaneously breaking when a new version changes something in the output format.

Reply:
I'll give that a try, I think I had an issue with environment resolving but that was at a phase of this project where I had lots of things broken at the same time. I might just need to look into how to get conda-forge to cross-compile/rebuld an arm64 build of an older doxygen version